spec/lib/govuk_error_spec.rb

```diff
@@ -19,4 +19,15 @@
       expect(Raven).to have_received(:capture_exception).with(StandardError.new, extra: { parameters: "Something" })
     end
   end
+
+  describe ".configure" do
+    it "configures Raven via the Configuration" do
+      mock_configuration = double("Configuration")
+      allow(GovukError::Configuration).to receive(:new) { mock_configuration }
+      expect(mock_configuration).to receive(:foo)
+      # rubocop:disable Style/SymbolProc
+      GovukError.configure { |config| config.foo }
```

Comment:
It doesn't seem too necessary to turn rubocop off here. Could just be `GovukError.configure(&:foo)`. If you want something more representative of the class you could stub a `foo=` method and then have a `config.foo = "something"` which won't need the rubocop disabling.